sdk/python/lib/pulumi/dynamic/dynamic.py

```diff
@@ -202,6 +202,8 @@ def delete(self, _id: str, _props: Any) -> None:
     def __init__(self) -> None:
         pass
 
+#https://github.com/python/mypy/issues/1102
```

Comment:
We have a general TODO style across our codebase of `TODO[pulumi/pulumi#1102]: Some description of the issue`.